pkg/operator/ceph/csi/spec.go

```diff
@@ -105,12 +107,13 @@ var (
 // manually challenging.
 var (
 	// image names
-	DefaultCSIPluginImage   = "quay.io/cephcsi/cephcsi:v3.2.0"
-	DefaultRegistrarImage   = "k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.0.1"
-	DefaultProvisionerImage = "k8s.gcr.io/sig-storage/csi-provisioner:v2.0.0"
-	DefaultAttacherImage    = "k8s.gcr.io/sig-storage/csi-attacher:v3.0.0"
-	DefaultSnapshotterImage = "k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.0"
-	DefaultResizerImage     = "k8s.gcr.io/sig-storage/csi-resizer:v1.0.0"
+	DefaultCSIPluginImage         = "quay.io/cephcsi/cephcsi:v3.2.0"
+	DefaultRegistrarImage         = "k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.0.1"
+	DefaultProvisionerImage       = "k8s.gcr.io/sig-storage/csi-provisioner:v2.0.0"
+	DefaultAttacherImage          = "k8s.gcr.io/sig-storage/csi-attacher:v3.0.0"
+	DefaultSnapshotterImage       = "k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.0"
+	DefaultResizerImage           = "k8s.gcr.io/sig-storage/csi-resizer:v1.0.0"
+	DefaultVolumeReplicationImage = "quay.io/kubestorage/volumereplication-operator:latest"
```

Comment:
Is there a versioned tag instead of `latest`? We could get broken if there is an update to the tag for a change we don't support.

Reply:
Yes, this PR should not get merged now, This is up for review. We can merge this one once we have replication operator release and cephcsi release. Adding DNM